content/docs/tasks/traffic-management/secure-ingress/index.md

```diff
@@ -470,6 +470,13 @@ they have valid values, according to the output of the following commands:
 
     `tls.crt` and `tls.key` should exist in the directory contents.
 
+1.  If you created the secret `istio-ingressgateway-certs`, but the key and the certificate are not loaded, kill the
```

Comment:
created the secret `istio-ingressgateway-certs` -> created the `istio-ingressgateway-certs` secret
kill -> delete